targeteditor/src/main/scala/explore/targeteditor/MagnitudeForm.scala

```diff
@@ -174,16 +180,24 @@ object MagnitudeForm {
             .setInitialStateFull(tableState)
 
           React.Fragment(
-            <.div(<.label("Magnitudes")),
-            Segment(
-              tableMaker.makeTable(
-                options = options,
-                data = props.magnitudes.toListOfViews(_.band).toJSArray,
-                headerCellFn = None,
-                tableClass = Css("ui very compact table"),
-                footer = footer
+            <.div(
+              <.label("Magnitudes"),
+              Segment(attached = SegmentAttached.Attached,
+                      compact = true,
+                      clazz = ExploreStyles.MagnitudesTableContainer
+              )(
+                tableMaker.makeTable(
+                  options = options,
+                  data = props.magnitudes.toListOfViews(_.band).toJSArray,
+                  headerCellFn = Some(c =>
+                    TableMaker
+                      .basicHeaderCellFn(Css.Empty)(c)
+                  ),
```

Comment:
`headerCellFn = Some(TableMaker.basicHeaderCellFn())` should also work. 

It looks like `scalajs-react-table` should use `Css.Empty` instead of `Css("")` for it's function default values. Not sure how I missed that.